webapp/components/wpt-results.html

```diff
@@ -151,6 +151,10 @@
   <script>
     /* global TestRunsBase */
     class WPTResults extends TestRunsBase {
+      static get UrlToResultsPath() {
+        return location => location.pathname.replace(/\/results(.+)(\/)?$/, '$1');
```

Comment:
This returns a functions, so why it worked puzzled me under I saw the `get` keyword: getter returns a functions which is invoked.

Does it need to be a getter, or can it just be a static function? That'd be more obvious.

Some possibly unintentional things about the regex:
* the `(\/)` group won't be used AFAICT. If the input ends with a slash, that slash will stell end up in the first group because of greedy matching. And the second group isn't referenced as $2, so need not be a group.
* Not sure if it's an invariant that `location` begins with "/results/" if this code is run, but this will map "/resultsabc/def" to "abc/def". If that's a bug, looks like `/\/results(\/.+)?$/` followed by slash normalization might be the way? (I do fairly frequently end up with double slashes in wpt.fyi URLs when pasting file paths into them, and it doesn't work, so normalizing that *somewhere* would be great.)

Reply:
Added some tests to cover the behaviour of the function.

As a result, the function can't be static (WCT can't do ECMA6, so far as I can tell).